### PR TITLE
added amax

### LINKED ISSUE #5789: amax

### PR DESCRIPTION
**Pull request by Viditagarwal7479**
- [ ] Close #5789

### CODE REVIEW

File: ivy/functional/frontends/torch/Tensor.py [hunk 1]
```diff
@@ -45,6 +45,9 @@ def float(self, memory_format=torch.preserve_format):
     def tan(self, *, out=None):
         return torch_frontend.tan(self.data, out=out)
 
+    def amax(self, dim=None, keepdim=False, *, out=None):
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/torch/Tensor.py:48]: Here, the implementation seems to be nice however the parameters for `Tensor.amax` are only dim and keepdim the instance method doesn't have an out argument `Tensor.amax(dim=None, keepdim=False) ` refer to documentation here [Tensor.amax](https://pytorch.org/docs/stable/generated/torch.Tensor.amax.html)

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey,
Thanks for contributing all the code looks excellent however here are a few changes to make.
And will go ahead and merge your PR provided the tests are passing.
Thanks:)

**Comment by Viditagarwal7479** [comment]
Sir, I think the tests are failing due to the part which I have not edited. So, as in discord, it was said not to worry about them.
Should I fix those files too?
Thanks
Vidit Agarwal

**Comment by hirwa-nshuti** [comment]
Hey,
You don't need to fix other failing tests just make the requested changes and I will go ahead merging your PR.
You can revert these last changes you made
Thanks.

**Comment by Viditagarwal7479** [comment]
Hey,
Thank you for your kind feedback I have made the requested changes and reverted my last change.
Thanks
Vidit

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py [hunk 2]
```diff
@@ -438,6 +438,39 @@ def test_torch_instance_tan(
...
+        available_dtypes=helpers.get_dtypes("valid", full=True),
+    ),
+)
+def test_torch_instance_amax(
+    dtype_x,
+    as_variable,
+    native_array,
+):
+    input_dtype, x = dtype_x
+    assume("bfloat16" not in input_dtype)
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:454]: Can you remove this line too

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
All looks great now one small change and I merge the PR

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py (hunk 2 above)
Comment (Viditagarwal7479) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py]: Sure

**Review by hirwa-nshuti — APPROVED** [review]
Looks good to me
Thanks for contributing🙂

**Comment by Viditagarwal7479** [comment]
Thanks for your kind feedback throughout, was waiting for such prompt feedback on my issues for quite a while. :smile:

**Pull request merged**